Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -728,7 +728,7 @@ ErrorOr<void> MainWidget::initialize_menubar(GUI::Window& window)
         "Layer via Copy", { Mod_Ctrl | Mod_Shift, Key_C }, g_icon_bag.new_layer, [&](auto&) {
             auto add_layer_success = current_image_editor()->add_new_layer_from_selection();
             if (add_layer_success.is_error()) {
-                GUI::MessageBox::show_error(&window, add_layer_success.release_error().string_literal());
+                GUI::MessageBox::show_error(&window, DeprecatedString::formatted("{}", add_layer_success.release_error()));
```

Comment:
Some of these currently use `MUST(String::formatted())` but here you're adding a `DeprecatedString::formatted()`? Could these use the new string already too?

Reply:
I've replaced these usages with `MUST(String::formatted(...))` and added an extra commit which ensures we are using this pattern for all error messages.